packages/schematics/angular/web-worker/index.ts

```diff
@@ -173,8 +173,15 @@ export default function (options: WebWorkerOptions): Rule {
 
     const needWebWorkerConfig = !projectTargetOptions.webWorkerTsConfig;
     if (needWebWorkerConfig) {
-      projectTargetOptions.webWorkerTsConfig =
-        `${root.endsWith('/') ? root : root + '/'}tsconfig.worker.json`;
+      const workerConfigPath = `${root.endsWith('/') ? root : root + '/'}tsconfig.worker.json`;
```

Comment:
are these paths always posix? how do we know to use forward slash?

Reply:
The convention is that they are always posix and relative to the `angular.json`. But we don't enforce it or error out if they are not posix. We do interpret them through the path abstraction in the devkit to convert them to system paths.

Reply:
I think after this PR, I’ll do another PR to refactor some logic in these schematics as there is some repetition among various schematics to generate a relative path.

Reply:
Most of our schematics rely on the fact that paths are posix.

Probably we can do some refactoring to support this. But i’d prefer if it was done in a separate PR.

Also, I’d like to extract some common methods which are repeated into various schematics so that they can be re-used.